test/sink/SafeSink-test.js

```diff
@@ -130,13 +127,13 @@ describe('SafeSink', function () {
 
     sink.error(0, new Error())
     sink.error(1, new Error())
-    expect(errorCalled).toBe(2)
+    eq(errorCalled, 2)
   })
 
   it('disable should return original sink', function () {
     var original = testSink(fail, fail, fail)
     var sink = new SafeSink(original)
 
-    expect(sink.disable()).toBe(original)
+    eq(sink.disable(), original)
```

Comment:
Hmm, this one is interesting.  At first, I thought it was a compelling case to use `is`: `original` could be any sink, and (unfortunately) since sinks tend to hold mutable state, it seems important to ensure `sink.disable()` returns a reference to `original`.

But using `is` may imply that it's ok for `SafeSink` to mutate fields of `original` in some way.  If it did, then `is` would pass.  Come to think of it, `eq` would pass as well!

Oddly enough, it seems like the most strict test would be something like:

```js
eq(sink.disable(), testSink(fail, fail, fail))
```

Is that crazy?  Maybe unnecessarily paranoid ... `eq` is probably fine :)  but wanted to see what you think.